silkrpc/types/call.hpp

```diff
@@ -42,7 +42,7 @@ struct Call {
         silkworm::Transaction txn{};
         txn.from = from;
         txn.to = to;
-        txn.gas_limit = gas.value_or(0);
+        txn.gas_limit = gas.value_or(30000000);
```

Comment:
Add missing unit test for this as suggested by Codecov

Reply:
few tests added